example/App.tsx

```diff
@@ -1,25 +1,31 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, View } from 'react-native';
-import exampleMapStyle from './exampleMapStyle.json';
+import { StatusBar } from "expo-status-bar";
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import exampleMapStyle from "./exampleMapStyle.json";
```

Comment:
We should all use same prettier config, I added some basic config in newest commit, you can rebase to use it so we don't do these unnecessary changes back and forth